sample-cf-install-values.yml

```diff
@@ -113,3 +113,11 @@ use_first_party_jwt_tokens: true
 
 load_balancer:
   enable: false
+
+#! optional configuration for using a private registry that has all cf-for-k8s images relocated to.
+#! see <docs/platform_operators/private-registry.md>
+private_registry:
+  add_image_pull_secrets: false
+  hostname: <registry_hostname>
+  username: <registry_username>
+  password: <registry_password>
```

Comment:
Could you please remove this change? We are still trying to figure out exactly how we want to position our sample values file, but the current feeling is that it does not need to include all available configuration options, as users can refer to the `config/values/00-values.yml` file for that.

Reply:
done